patch/cisco-hwmon-pmbus_core-pec-support-check.patch

```diff
@@ -0,0 +1,39 @@
+From ca2339d26b3473aa4f85006c5caff8eb01fd5435 Mon Sep 17 00:00:00 2001
+From: Madhava Reddy Siddareddygari <msiddare@cisco.com>
+Date: Wed, 26 May 2021 11:25:14 -0700
+Subject: [PATCH] hwmon: (pmbus_core) Do not enable PEC if adapter doesn't
+ support it
+
+Added code to check if i2c bus has capability to support PEC [ Packet Error Correction ] and enable it only if both controller and bus support it
+
+I2C_FUNC_SMBUS_PEC can be used to check PEC capability
```

Comment:
Please add a dot/period at the end of sentences.

Reply:
Done